src/App.js

```diff
@@ -1,25 +1,66 @@
-import logo from './logo.svg';
-import './App.css';
+import NavBar from "./components/navbar";
+import Counters from "./components/counters";
+import "./App.css";
+import React from "react";
+import { Component } from "react";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  state = {
+    counters: [
+      { id: 1, value: 0 },
+      { id: 2, value: 0 },
+      { id: 3, value: 0 },
+      { id: 4, value: 0 },
+    ],
+  };
+
+  constructor() {
+    super();
+    this.handleDelete.bind(this);
+    this.handleReset.bind(this);
+    this.handleIncrement.bind(this);
+  }
+
+  handleReset = () => {
+    const counters = this.state.counters.map((c) => {
+      c.value = 0;
+      return c;
+    });
+    this.setState({ counters });
+  };
+
+  handleDelete = (counterId) => {
+    const counters = this.state.counters.filter((c) => c.id !== counterId);
+    this.setState({ counters });
+    // This is same as this.setState({ counters: counters })
+  };
+
+  handleIncrement = (counter) => {
+    // Cloning array, but the objects in it are the same
+    const counters = [...this.state.counters];
+    const index = counters.indexOf(counter);
+    counters[index] = { ...counter };
+    counters[index].value++;
```

Comment:
@Orangestripything also fyi you can leave comments on specific lines of code like dis